### PR TITLE
[Microsoft.ApiManagement][2019-12-01] Content management resources

### PR DESCRIPTION
**Pull request by azaslonov**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories. 
  - adding/removing APIs.
  - adding/removing properties.
  - adding/removing API-version. 
  - adding a new service in Azure.

<i>Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.</i>

- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by raych1** [comment]
@azaslonov , please fix the CI check failures.

**Comment by azaslonov** [comment]
@raych1 all is green now. Thanks for looking into this.

**Review by majastrz — COMMENTED** [review]
As per https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md, evolutionary changes should be added in a new API version.

**Comment by solankisamir** [comment]
> As per https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md, evolutionary changes should be added in a new API version.

@majastrz this is ARM KPI Swagger completeness. The API is being used by customers in 2019-12-01 and we missed to document it.

GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.APIMANAGEMENT/SERVICE/CONTENTTYPES/CONTENTITEMS/

You can refer to task - https://msazure.visualstudio.com/Unified%20Platform%20KPIs/_workitems/edit/3598671

**Comment by majastrz** [comment]
> > As per https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md, evolutionary changes should be added in a new API version.
> 
> @majastrz this is ARM KPI Swagger completeness. The API is being used by customers in 2019-12-01 and we missed to document it.
> 
> GET/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.APIMANAGEMENT/SERVICE/CONTENTTYPES/CONTENTITEMS/
> 
> You can refer to task - https://msazure.visualstudio.com/Unified%20Platform%20KPIs/_workitems/edit/3598671
> /cc @azaslonov

Makes sense. This week's ARM reviewer (@chiragg4u) will take a look.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/examples/ApiManagementGetContentType.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    "description": "Reference to page content document.",
+                    "type": "string"
+                  }
+                },
+                "additionalProperties": false,
+                "required": [
+                  "title",
+                  "permalink",
+                  "documentId"
+                ]
```
Comment (chiragg4u) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/examples/ApiManagementGetContentType.json:64]: This looks like the swagger definition for this object will this be returned to the user?

Comment (azaslonov) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/examples/ApiManagementGetContentType.json:64]: Yes, this is the schema for ContentItem defined by a customer.

File: specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/definitions.json [hunk 2]
```diff
@@ -5565,6 +5565,100 @@
...
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Content type contract details."
+    },
+    "ContentItemContractProperties": {
+      "properties": {},
+      "additionalProperties": true
```
Comment (chiragg4u) [inline specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/definitions.json:5661]: The definition is open, how will SDK/clients be able to parse this and make sense of the response?

Comment (azaslonov) [reply specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2019-12-01/definitions.json:5661]: I assume they will parse it as a string, they don't need to make sense of it. It is basically content.

**Review by chiragg4u — APPROVED** [review]
Minor comments based on responses, changes are approved.

**Review by solankisamir — APPROVED** [review]

**Pull request merged**